_posts/molecules/2018-10-01-autocomplete.html

```diff
@@ -3,35 +3,21 @@
 title:  Autocomplete
 category: molecules
 description: Whenever a field has the ability to autocomplete its value, we should add a hint in form of placeholder and 
-             also an icon on the left that shows a spinner while loading from remote
+             also an icon on the left that shows a spinner while loading from remote.
+             OBS provides an autocomplete parti, including javascript for handling the autocomplete request and spinner. For
+             further customization of autocomplete events please refer to the jquery autocomplete documentation (https://jqueryui.com/autocomplete/).
 ---
 
 %h5 Input with autocompletion
 
-.form-group#form-group-id
-    %label{ for: 'input-id' }
-        Label:
-    .input-group
-        .input-group-prepend
-            %span.input-group-text#search-icon-id
-                %i.fa.fa-search
-                %i.fas.fa-spinner.fa-spin.d-none
-        %input.form-control#input-id{ type: 'text', placeholder: 'Type to autocomplete...' }
-
-:javascript
-    $('#input_id').autocomplete({
-        appendTo: '#form-group-id',
-        source: 'http://localhost?url_to_remote',
-        search: function(event, ui) {
-            var icon = $('#search-icon-id i:first-child');
-            icon.addClass('d-none');
-            icon.next().removeClass('d-none');
-        },
-        response: function(event, ui) {
-            var icon = $('#search-icon-id i:first-child');
-            icon.removeClass('d-none');
-            icon.next().addClass('d-none');
-        },
-        minLength: 2
-    });
-      
+// Below is the content of the autocomplete partial. Render it in your view, e.g.
+// "render partial: 'webui/autocomplete', locals: { html_id: 'linked_project', label: 'Original project name', source: autocomplete_projects_path }"
+.form-group.ui-front
+  %label{ for: 'input-id' }
+    Label:
+  .input-group
+    .input-group-prepend
+      %span.input-group-text
+        %i.fas.fa-search
+    %input.form-control#input-id{ class: 'obs-autocomplete form-control', data: { source: '/path/to/autocomplete/data' },
```

Comment:
The classes should be added directly (so not inside `{}`). `form-control` is also added twice.

Reply:
Still valid, I added a suggestion.